File: Patterns/02. Rectangle Patterns/14. Rectangle Decreasing Pattern/rdp.java
import java.util.*;
public class rdp{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter the Length : ");
        int l = sc.nextInt();

        System.out.print("Enter the Breadth : ");
        int b = sc.nextInt();

        for(int i=1; i<=l; i++){
            for(int j=b; j>=1; j--){ // we are printing j, so don't need to change the order of i
                System.out.print(j + " ");
            }
            System.out.println();
        }
        sc.close();
    }
}

/*
Output

7 6 5 4 3 2 1 
7 6 5 4 3 2 1 
7 6 5 4 3 2 1 
7 6 5 4 3 2 1 
7 6 5 4 3 2 1 
*/

File: Patterns/02. Rectangle Patterns/14. Rectangle Decreasing Pattern/rdp.py
l = int(input("Enter the Length : "))
b = int(input("Enter the Breadth : "))

for i in range(1,l+1):
    for j in range(b,0,-1): # we are printing j, so don't need to change the order of i
            print(j, end=' ')
    print()

'''
Output:

7 6 5 4 3 2 1 
7 6 5 4 3 2 1 
7 6 5 4 3 2 1 
7 6 5 4 3 2 1 
7 6 5 4 3 2 1 
'''